### PR TITLE
Update documentation to include the now randomized backup schedules

### PR DESCRIPTION
**Pull request by glrf**

### CODE REVIEW

File: docs/modules/ROOT/pages/references/crds.adoc [hunk 1]
```diff
@@ -40,6 +40,24 @@ This is a https://github.com/elastic/crd-ref-docs[generated] API documentation.
 |===
 
 
+[id="{anchor_prefix}-github-com-vshn-component-appcat-apis-v1-localobjectreference"]
+=== LocalObjectReference 
+
+LocalObjectReference contains enough information to let you locate the referenced object inside the same namespace.
```
Comment (zugao) [inline docs/modules/ROOT/pages/references/crds.adoc:46]: I would remove `enough`

File: docs/modules/ROOT/pages/vshn-managed/postgresql/backup.adoc [hunk 2]
```diff
@@ -1,6 +1,8 @@
 = Backups
 
-Backups are enabled by default and your PostgreSQL instance  has the following backup configuration:
+Backups are enabled by default the PostgreSQL instance will be backed up daily at a randomly chosen time between 22:00 and 4:00.
```
Comment (zugao) [inline docs/modules/ROOT/pages/vshn-managed/postgresql/backup.adoc:3]:
```suggestion
Backups are enabled by default. The PostgreSQL instance will be backed up daily at a randomly chosen time between 22:00 and 4:00.
```

**Review by zugao — APPROVED** [review]
LGTM, some small things.

File: docs/modules/ROOT/pages/references/crds.adoc (hunk 1 above)
Comment (glrf) [reply docs/modules/ROOT/pages/references/crds.adoc:46]: This is auto generated from the CRD and was copied directly from upstream k8s comment on `LocalObjectRefernce`. Is it OK for you if I leave that?

Comment (zugao) [reply docs/modules/ROOT/pages/references/crds.adoc:46]: no problem

**Pull request merged**